.travis.yml

```diff
@@ -4,13 +4,12 @@ language: java
 before_install:
   - mkdir ci-dependencies
   - cd ci-dependencies
-  - curl -O https://openskywalking.github.io/skywalking-ci-assist/jars/ojdbc14-10.2.0.4.0.jar
+  - curl -O https://github.com/SkyAPM/ci-assist/blob/master/jars/ojdbc14-10.2.0.4.0.jar
```

Comment:
https://github.com/SkyAPM/ci-assist/blob/master/jars/ojdbc14-10.2.0.4.0.jar?raw=true

This is the real download URL, if you want to download them from GitHub

Reply:
Or this, https://skyapm.github.io/ci-assist/jars/ojdbc14-10.2.0.4.0.jar